apps/vendor/models.py

```diff
@@ -0,0 +1,168 @@
+    gst_category_id = models.ForeignKey(GstCategories, on_delete=models.CASCADE, null=True, default=None, db_column='gst_category_id')
+    gst_suspend = models.BooleanField(null=True, default=None)
+    TAX_TYPE_CHOICES = [('inclusive', 'Inclusive'),
+                        ('exclusive', 'Exclusive')
+                        ]
+    tax_type = models.CharField(max_length=20, choices=TAX_TYPE_CHOICES , default= 'Inclusive')
+    distance = models.DecimalField(max_digits=18, decimal_places=2, null=True, default=None)
+    tds_on_gst_applicable = models.BooleanField(null=True, default=None)
+    tds_applicable = models.BooleanField(null=True, default=None)
+    website = models.URLField(max_length=255, null=True, default=None)
+    facebook = models.URLField(max_length=255, null=True, default=None)
+    skype = models.URLField(max_length=255, null=True, default=None)
+    twitter = models.URLField(max_length=255, null=True, default=None)
+    linked_in = models.URLField(max_length=255, null=True, default=None)
+    payment_term_id = models.ForeignKey(VendorPaymentTerms,on_delete=models.CASCADE, null=True, default=None, db_column='payment_term_id')
+    price_category_id = models.ForeignKey(PriceCategories, on_delete=models.CASCADE, null=True, default=None, db_column='price_category_id')
+    vendor_agent_id = models.ForeignKey(VendorAgent, on_delete=models.CASCADE, null=True, default=None, db_column='vendor_agent_id')
+    transporter_id = models.ForeignKey(Transporters, on_delete=models.CASCADE, null=True, default=None, db_column='transporter_id')
+    credit_limit = models.DecimalField(max_digits=18, decimal_places=2, null=True, default=None)
+    max_credit_days = models.IntegerField(null=True, default=None)
+    interest_rate_yearly = models.DecimalField(max_digits=18, decimal_places=2, null=True, default=None)
+    rtgs_ifsc_code = models.CharField(max_length=255, null=True, default=None)
+    accounts_number = models.CharField(max_length=255, null=True, default=None)
+    bank_name = models.CharField(max_length=255, null=True, default=None)
+    branch = models.CharField(max_length=255, null=True, default=None)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    
+    def __str__(self):
+        return f'{self.vendor_id}_{self.name} ({self.code})'
+
+    class Meta:
+        db_table = 'vendor'
+
+
+class VendorAttachment(models.Model):
+    attachment_id = models.AutoField(primary_key=True)
+    vendor_id = models.ForeignKey(Vendor, on_delete=models.CASCADE, null=True, default=None, db_column='vendor_id')
+    attachment_name = models.CharField(max_length=255, null=True, default=None)
+    attachment_path = models.CharField(max_length=255, null=True, default=None)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        db_table = 'vendor_attachments'  # Set custom table name
```

Comment:
db_table value should come from utils_variables.py
/

Reply:
ok